### DISCUSSION TITLE
strongDMI question

### DISCUSSION THREAD

**Discussion by Hinaichigo** [vgstation-coders/vgstation13#31795] [Q&A] [answered]
@kane-f I'm trying to get strongDMI installed with krita but I encounter the following error. I already ran pip install byondtools but maybe it didn't work somehow. If you could please let me know the best way to fix this I'd greatly appreciate it.

![kritathing](https://user-images.githubusercontent.com/7112773/148298890-5fa477db-22d9-4276-b784-73f24770b457.png)

**Comment by ghost** [comment, marked as answer]
oh sorry for the late response, N3X15 said he was gonna rework byondtools so i dunno if the pip version is good to use for now.
also try making sure pip doesnt install if locally maybe

**Reply by Hinaichigo** [reply]
Thanks.
